README.md

```diff
@@ -1 +1,103 @@
+    
+   **How it works**:
+   - The workflow generates PRs against each active release branch corresponding to an RHDH release.
+   - For each release branch, it checks for plugin versions compatible with the Backstage version that the release supports.
+   - If no compatible version is found, no PR is generated for that plugin on that branch.
+
+### Manually Creating a PR
+
+You can also create PRs manually by referencing existing examples: [View PR examples](https://github.com/redhat-developer/rhdh-plugin-export-overlays/pulls/app%2Fgithub-actions)
+
+To add a new plugin:
+
+1. Create a new workspace in the overlay repository.
+2. Add a `plugins-list.yaml` file that lists all plugins included in the target workspace of the source repository. ([See example](https://github.com/redhat-developer/rhdh-plugin-export-overlays/blob/12acb71a1febc5567c4d12c6a28c0a11ed489273/workspaces/adoption-insights/plugins-list.yaml))
+3. Add a `source.json` file with the following fields ([See example](https://github.com/redhat-developer/rhdh-plugin-export-overlays/blob/12acb71a1febc5567c4d12c6a28c0a11ed489273/workspaces/adoption-insights/source.json))
+:
+   - `repo`: URL of the source repository  (only `https://github.com/xxx` URLs are supported for now)
+   - `repo-ref`: Specific tag or commit for the target plugin/workspace version  
+   - `repo-flat`:  
+     - `false` if the plugins are inside a workspace (e.g., `backstage/community-plugins`)  
+     - `true` if the plugins are at the root level (e.g., `backstage/backstage`)
+
+### 3. Add Additional Dynamic Plugin Export Information (If Needed)
+
+Sometimes, additional configuration is required in the PR:
+
+- **Frontend plugins** may need:
+   - `app-config.dynamic.yaml` (Eg: [techdocs plugin](https://github.com/redhat-developer/rhdh-plugin-export-overlays/blob/release-1.5/workspaces/backstage/plugins/techdocs/app-config.dynamic.yaml))
+   - `scalprum-config.json` (Eg: [api-docs-module-protoc-gen-doc plugin](https://github.com/redhat-developer/rhdh-plugin-export-overlays/blob/release-1.5/workspaces/backstage/plugins/api-docs-module-protoc-gen-doc/scalprum-config.json))
+
+- **Any plugin** may need:
+   - Overlay source files in an `overlay` directory  
+  (e.g., [`api-docs-module-protoc-gen-doc`](https://github.com/redhat-developer/rhdh-plugin-export-overlays/tree/release-1.5/workspaces/backstage/plugins/api-docs-module-protoc-gen-doc/overlay))
+  - A patch file at the root of the workspace to modify plugin source code during the packaging process. (Example: PR [#792](https://github.com/redhat-developer/rhdh-plugin-export-overlays/pull/792/files#diff-0b648cbca6f87e11f78832c10ac9cc789235938e944c499eb275fd8788e18ef8))
+
+> **Overlay vs. Patch**  
+> - **Overlay**: Replaces or adds entire files during the packaging process.  
+> - **Patch**: Applies precise, line-by-line changes to existing source files.  
+
+
+To add this:
```

Comment:
When adding a patch, you don't need to create the plugins/<plugin-name> folder since the patch file is added at the workspace (== monorepo) root.